src/dune_pkg/lock_dir.mli

```diff
@@ -52,6 +52,7 @@ type t =
 val remove_locs : t -> t
 val equal : t -> t -> bool
 val to_dyn : t -> Dyn.t
+val pp_packages : t -> 'a Pp.t list
```

Comment:
Can you do without this function for now?

1. The type of this function is wrong. `pp_packages` should return `Pp.t` based on the name.
2. We don't need pretty printing functions like this in the dune_pkg API